scene/animation/animation_mixer.h

```diff
@@ -143,6 +143,8 @@ class AnimationMixer : public Node {
 		Object *object = nullptr;
 		ObjectID object_id;
 		real_t total_weight = 0.0;
+
+		virtual ~TrackCache(){};
```

Comment:
Style nit, not critical.
```suggestion
		virtual ~TrackCache() {}
```